Makefile.docs

```diff
@@ -0,0 +1,90 @@
+# user defined variables
+LANGUAGES ?=
+
+# python specific variables
+PYTHON_MAIN_DIR ?= src
+
+# provided by docsrv
+DESTINATION_FOLDER ?=
+SHARED_FOLDER ?=
+BASE_URL ?=
+REPOSITORY_OWNER ?=
+REPOSITORY ?=
+VERSION_NAME ?=
+
+API_DOCS_FOLDER := "$(DESTINATION_FOLDER)/api"
+CI_FOLDER := "$(SHARED_FOLDER)/ci"
+
+docs: docs-site docs-api
+
+install-ci:
+	@if [ ! -d "$(CI_FOLDER)" ]; then \
+		git clone https://github.com/src-d/ci $(CI_FOLDER); \
```

Comment:
You never call this makefile directly. You copy it from the CI folder to `.` and then include it. Anyway, I'm gonna remove this task, it's no longer needed. On docsrv is installed in the init script and locally it is installed in the concrete makefile.